components/butterfly/protocols/swim.proto

```diff
@@ -51,6 +51,7 @@ message Service {
   optional string pkg = 9;
   optional bytes cfg = 10;
   optional SysInfo sys = 12;
+  optional string channel = 13;
```

Comment:
I agree with leaving it out of the gossip if it is not required there.
We should have a clear segregation of layers and adapters between them to serve information in a easily consumable format to the consuming layer.